### PR TITLE
NF: Handle situations where CSD auto-response function cannot complete.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by 36000** [comment]
Should we merge #774  and then rebase this on that, so that Sendy can test this?

File: AFQ/models/csd.py [hunk 1]
```diff
@@ -19,6 +19,9 @@
 __all__ = ["fit_csd"]
 
 
+class CsdNanResponseError(Exception):
+
```
Comment (36000) [inline AFQ/models/csd.py:23]: You need a `    pass` here

**Comment by arokem** [comment]
If #774 is ready to go, then yes. But it looks like that one is waiting on #764 to be merged?

**Comment by arokem** [comment]
Looks like this is not fundamentally broken. @SendyCaffarra : do you want to pull this one down onto the cluster and run it? It should raise a (relatively) informative error for those cases where you've been getting CSD-related errors.

**Comment by arokem** [comment]
Rebased on top of current master. Do we know if it does what it's supposed to? 

@SendyCaffarra : did you get these errors happening again? Were you able to figure out which subject they came from using this error?

**Comment by SendyCaffarra** [comment]
Yes!
This time I got informative errors and I could detect the problematic subject.

**Comment by arokem** [comment]
Great! @36000 : should we merge this and then cut a release early next week?

**Pull request merged**